dev-cluster/deploy.sh

```diff
@@ -3,13 +3,22 @@ set -e
 
 cd "${0:A:h}"
 
+if [[ $(uname) == "Darwin" ]]; then
+  export K3D_FIX_DNS=0
+  if command -v colima >/dev/null 2>&1; then
+    if colima status >/dev/null 2>&1; then
+      export DOCKER_CONTEXT='colima'
+    fi
+  fi
+fi
+
 k3d cluster delete exampleapp || true
 k3d registry delete k3d-exampleapp-registry.local.lncd.pl || true
+rm .terraform.lock.hcl || true
 rm *.tfstate* || true
```

Comment:
```suggestion
rm -f .terraform.lock.hcl *.tfstate*
```

Let's move `set -e` from the top just above `helm repo update`, there are so many `|| true` that having it at the top defeats the point.